books/free-programming-books-subjects.md

```diff
@@ -308,7 +308,7 @@ Books that cover a specific programming language can be found in the [BY PROGRAM
 * [Control and Embedded Systems](http://www.learn-c.com) (HTML)
 * [Discovering the STM32 Microcontroller](http://www.cs.indiana.edu/~geobrown/book.pdf) (PDF) (CC BY-NC-SA)
 * [First Steps with Embedded Systems](https://www.phaedsys.com/principals/bytecraft/bytecraftdata/bcfirststeps.pdf) - Byte Craft Limited (PDF)
-* [Introduction to Embedded Systems, Second Edition](https://ptolemy.berkeley.edu/books/leeseshia/releases/LeeSeshia_DigitalV2_2.pdf) - Edward Ashford Lee, Sanjit Arunkumar Seshia (PDF)
+* [Introduction to Embedded Systems, Second Edition](https://ptolemy.berkeley.edu/books/leeseshia/releases/LeeSeshia_DigitalV2_2.pdf) - Edward Ashford Lee, Sanjit Arunkumar Seshia (PDF) (CC EY-NC-ND)
```

Comment:
* EY -> BY
* Please edit your PR description to indicate how/where you found the license notation

Reply:
Updated the description of the PR, can you do a quick verification?

Reply:
still wrong

Reply:
Updated description and edited license for the book, can you do a quick review again?